test/system/posts_test.rb

```diff
@@ -0,0 +1,12 @@
+require "application_system_test_case"
+
+class PostsTest < ApplicationSystemTestCase
+  test "adding a post" do
+    visit root_path
+    sign_up
+    click_link "Post your bin"
+    fill_in "Image url", with: "https://i.imgur.com/C5E1ya6.jpg"
+    click_button "Post"
+    assert_text "Latest Posts"
```

Comment:
A stronger assertion could be to assert that the image that we added appears on the page